skyrim-platform/src/platform_se/skyrim_platform/TextsCollection.cpp

```diff
@@ -44,6 +45,36 @@ void TextsCollection::SetTextColor(int textId, std::array<double, 4> color)
   texts.at(textId).color = color;
 }
 
+void TextsCollection::SetTextFont(int textId, std::string name)
+{
+  texts.at(textId).fontName = name;
+}
+
+void TextsCollection::SetTextRotation(int textId, float rotation)
+{
+  texts.at(textId).rotation = rotation;
+}
+
+void TextsCollection::SetTextSize(int textId, float size)
+{
+  texts.at(textId).size = size;
+}
+
+void TextsCollection::SetTextEffect(int textId, int effect)
+{
+  texts.at(textId).effects = static_cast<DirectX::SpriteEffects>(effect);
```

Comment:
Static casting to a enum seems not safe... Let's at least add all the possible values to some JS/TS constant. I.e. we probably should make some kind of constant so we could write like this:

`setSpriteEffect(textId, sp.SpriteEffects.SomeEffect1 | sp.SpriteEffects.SomeEffect2)`